packages/babel-plugin-import-jsx-pragma/README.md

```diff
@@ -38,7 +38,7 @@ _Note:_ `@wordpress/babel-plugin-import-jsx-pragma` is included in `@wordpress/b
 
 As the `@babel/plugin-transform-react-jsx` plugin offers options to customize the `pragma` to which the transform references, there are equivalent options to assign for customizing the imports generated.
 
-For example, if you are using the `@wordpress/element` package, you may want to use the following configuration:
+For example, if you are using the `React` package, you may want to use the following configuration:
```

Comment:
As a followup we should consider whether we need this Babel plugin at all. It certainly made sense 5 years ago, but newer versions of React and Babel support the "automatic runtime", which also adds the imports automatically. They support auto-importing from the `react/jsx-runtime` package, but maybe we can configure the transform to use the "classic" import from `react`.

Reply:
It looks like this PR is only the starting point. We should get feedback soon on whether the transition is smooth for everyone. It should be. Eventually, we can deprecate the plugin as React supports the automatic runtime, which wasn't the case when this plugin was introduced.